libs/wingsdk/src/target-awscdk/secret.ts

```diff
@@ -12,14 +15,14 @@ import { IInflightHost } from "../std";
  * @inflight `@winglang/sdk.cloud.ISecretClient`
  */
 export class Secret extends cloud.Secret {
-  private readonly secret: CdkSecret;
+  private readonly secret: ICdkSecret;
 
   constructor(scope: Construct, id: string, props: cloud.SecretProps = {}) {
     super(scope, id, props);
 
-    this.secret = new CdkSecret(this, "Default", {
-      secretName: props.name,
-    });
+    this.secret = props.name
+      ? CdkSecret.fromSecretNameV2(this, "Default", props.name)
+      : new CdkSecret(this, "Default");
```

Comment:
(not blocking) I wonder if we this is something we should mention in the docstring for `cloud.Secret`, or if this is something we could document in a dedicated README / conceptual docs page about the `cloud.Secret` resource?

Reply:
Ah I forgot we have an issue related to this https://github.com/winglang/wing/issues/1438 - @jogold is this something you'd like to try? (Only if it interests you of course 🙂)

Reply:
Maybe... not sure 🙂  will let know and assign myself if I do.